package.json

```diff
@@ -45,12 +50,17 @@
     "grunt-retire": "0.3.12",
     "grunt-template-jasmine-requirejs": "0.2.3",
     "handlebars": "4.0.5",
-    "lodash": "4.1.0",
+    "jquery": "1.12.1",
+    "jquery.cookie": "1.4.1",
+    "json-loader": "0.5.4",
     "moment": "2.12.0",
     "normalize-url": "1.4.1",
     "open": "0.0.5",
+    "q": "1.4.1",
     "request-promise": "2.0.0",
-    "semver": "5.1.0"
+    "semver": "5.1.0",
+    "underscore": "1.8.3",
```

Comment:
I prefer lodash as well, but it doesn't make sense to include it as a dependency if we're not using any of the features not in underscore.